plrust/src/error.rs

```diff
@@ -32,6 +30,6 @@ pub(crate) enum PlRustError {
     ParsingCodeBlock(syn::Error),
     #[error("Parsing error at span `{:?}`", .0.span())]
     Parse(#[from] syn::Error),
-    #[error("No plrust.plrust_proc entry for ({0}, {1})")]
-    NoProcEntry(pgx::pg_sys::Oid, CompilationTarget),
+    #[error("plrust.plrust_proc.so bytes are NULL")]
+    NullPlRustProcSharedLibraryBytes,
```

Comment:
Does this mean they are all `\0`s or that it is a pointer to address 0x0?

Reply:
It means the datum is an SQL NULL.

Reply:
The column itself is declared NOT NULL so this would only happen if Postgres broke

Reply:
Ah okay.
In such a case we should probably simply panic, but
```suggestion
    #[error("plrust.plrust_proc.so is an SQL NULL despite being NOT NULL")]
    NullPlRustProcSharedLibraryBytes,
```